### DISCUSSION TITLE
Is it possible to lower the weight or hide results from specific domains?

### DISCUSSION THREAD

**Discussion by BBernerz** [searxng/searxng#2219] [Q&A]
Hey there SearXNG community. I am sick and tired of search results giving fake news, fact checkers, and all the other BS out there. I have compiled a list of untrustworthy domains that share fake news. I would like search results to exclude these websites. For example, lets say I wanted to block YouTube results, therefor increasing the likelihood of Rumble or BitChute results from appearing instead. Is it possible to blacklist domains from results, or adjust weights of specific domains to increase or decrease the likelihood of their results appearing?

Thanks!

**Comment by justyns** [comment]
Did you end up posting your list anywhere?

If you haven't found it yet, there is a built-in hostname_replace plugin that can remove results from domains that certain regex.   This has some examples: https://github.com/searxng/searxng/discussions/970

**Comment by sjehuda** [comment]
@BBernerz I've made a [userscript](https://openuserjs.org/scripts/sjehuda/SearX_Results_Filter) that does this task for you. You need to have a Userscript manager installed in order to use this userscript.

Results can be filtered by `string` and `hash code`.  Hash code is useful when listing XXX websites without mention of their host names.

This userscript has been tested with link, image and video search results.

See also #1177

EDIT: @masterofobzene **has already [made](https://github.com/searxng/searxng/issues/1258#issuecomment-1172940391) an extension that does this task. See [SearchBlocklist](https://github.com/masterofobzene/SearchBlocklist).**

**Reply by sjehuda** [reply]
All is left is for users to add remove hashes or hostnames to filter.

To add hostnames or hash code, you can point your cursor on a wanted link in the results or visit that website.

I think I've done most of the work on this script, but I should probably make a Greasemonkey config interface as it is more accessible and easier.
